specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2016-05-01/Backups.json

```diff
@@ -0,0 +1,301 @@
+                        "description": "OK"
+                    },
+                    "202": {
+                        "description": "ACCEPTED"
+                    },
+                    "404": {
+                        "description": "NOT FOUND"
+                    }
+                },
+                "x-ms-long-running-operation": true
+            }
+        }
+    },
+    "definitions": {
+        "Backup": {
+            "description": "Information related to a backup.",
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "description": "Properties for a backup.",
+                    "$ref": "#/definitions/BackupModel",
+                    "x-ms-client-flatten": true
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "Backup.json#/definitions/Resource"
+                }
+            ]
+        },
+        "BackupModel": {
+            "description": "Properties for a backup.",
+            "properties": {
+                "backupInfo": {
+                    "description": "Holds information for a backup.",
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/BackupInfo"
+                }
+            }
+        },
+        "BackupInfo": {
+            "type": "object",
+            "description": "Holds information for a backup.",
+            "properties": {
+                "backupDataVersion": {
+                    "description": "Version of the backup data.",
+                    "type": "string"
+                },
+                "backupId": {
+                    "description": "Unique GUID for the backup.",
+                    "type": "string"
+                },
+                "roleStatus": {
+                    "description": "Backup status for each role.",
+                    "$ref": "#/definitions/RoleOperationStatusList"
+                },
+                "status": {
+                    "description": "Current status of the backup.",
+                    "$ref": "#/definitions/OperationStatus"
+                },
+                "createdDateTime": {
+                    "description": "Creation time of the backup.",
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "timeTakenToCreate": {
+                    "description": "Duration to create the backup.",
+                    "type": "string"
+                },
+                "deploymentID": {
+                    "description": "Deployment Id of the stamp.",
+                    "type": "string"
+                },
+                "stampVersion": {
+                    "description": "Current version.",
+                    "type": "string"
+                },
+                "oemVersion": {
+                    "description": "OEM version.",
+                    "type": "string"
+                }
+            }
```

Comment:
Is it possible to mark properties  of `BackupInfo` as readonly? i don't see it is used in create or update so thought it is eligible to be marked as readonly

Reply:
Done.